client/src/components/campaignCreatorComponents/CampaignCreator.tsx

```diff
@@ -0,0 +1,153 @@
+import PillarOptions from './PillarOptions';
+import OptionListCC from './OptionListCC';
+import ccData from '../../static_data/campaign_creator.json';
+import RadioButtonListCC from './RadioButtonListCC';
+import { Button, Heading, Flex, Separator } from '@radix-ui/themes';
+import { TLSelect } from '../primitiveComponents/Primitives';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+interface CampaignCreatorProps {
+    campaignSettings: Record<string, boolean>;
+    setCampaignSettings: React.Dispatch<React.SetStateAction<{}>>;
+}
+
+const CampaignCreator = ({ campaignSettings, setCampaignSettings }: CampaignCreatorProps) => {
+    const navigate = useNavigate();
+
+    useEffect(() => console.log('checkbox status', campaignSettings), [campaignSettings]);
```

Comment:
Remember to remove these when not needed

Reply:
removed